src/app/quiz/components/checkboxChoiceInput.js

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import Checkbox from '@mui/material/Checkbox';
+import FormGroup from '@mui/material/FormGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import FormControl from '@mui/material/FormControl';
+import FormLabel from '@mui/material/FormLabel';
+import Grid from "@mui/material/Grid";
+import Button from "@mui/material/Button";
+import {SvgIcon} from "@mui/material";
+import Container from "@mui/material/Container";
+import Box from "@mui/material/Box";
+import useSWR from "swr"
+import { useState } from 'react';
+
+const post_fetcher = (...args) => fetch(...args).then(res => res.json())
+
+function ChoiceVariantButton({ choice, question }) {
```

Comment:
It is not a button, actually, it is a checkbox, which should be reflected in naming. Otherwise one could think, that it is a button really.